app/code/Magento/Checkout/Controller/Sidebar/UpdateItemQty.php

```diff
@@ -62,11 +62,16 @@ public function __construct(
 
     /**
      * @return $this
+     * @throws LocalizedException
      */
     public function execute()
     {
+        if (!$this->getRequest()->isPost()) {
+            throw new LocalizedException(__('Wrong request.'));
+        }
+
```

Comment:
Please revert this change and add add to the class that it implements `Magento\Framework\App\Action\HttpPostActionInterface` interface
```suggestion
```

Also, please cover your change with any type of automated test

Reply:
I believe that unt tests is enough for this case